src/app/settings/panel-settings/components/edit-panel-layout-configuration/edit-panel-layout-configuration.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core'
+import { PanelLayoutConfiguration } from '../../models/panel-layout-configuration'
+import { FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms'
+
+@Component({
+  selector: 'sofie-edit-panel-layout-configuration',
+  templateUrl: './edit-panel-layout-configuration.component.html',
+  styleUrls: ['./edit-panel-layout-configuration.component.scss'],
+})
+export class EditPanelLayoutConfigurationComponent implements OnInit {
+  @Input()
+  public panelLayoutConfiguration: PanelLayoutConfiguration
+
+  @Output()
+  public onSave: EventEmitter<PanelLayoutConfiguration> = new EventEmitter()
+  @Output()
+  public onCancel: EventEmitter<void> = new EventEmitter()
```

Comment:
```suggestion
  @Input() public panelLayoutConfiguration: PanelLayoutConfiguration

  @Output() public onSave: EventEmitter<PanelLayoutConfiguration> = new EventEmitter()
  @Output() public onCancel: EventEmitter<void> = new EventEmitter()
```

Just a suggestion :)